tests/test_utils/test_context_utils.py

```diff
@@ -95,22 +95,32 @@ def test_merge_contexts_base():
     ("imp", "result"),
     [
         ("linkml:types", f"C:\\temp\\linkml_model\\model\\schema{os.sep}types"),
-        ("ex_file", "/tmp/example/schema"),
+        ("ex_file", "/tmp/example/schema"),  # noqa: S108
         ("_types", "https://w3id.org/linkml/types"),
     ],
 )
-def test_map_import(imp, result):
+def test_map_import(imp: str, result: str) -> None:
     importmap = {
         "linkml:": "C:\\temp\\linkml_model\\model\\schema",
-        "ex_file": "/tmp/example/schema",
+        "ex_file": "/tmp/example/schema",  # noqa: S108
         "_types": "linkml:types",
     }
 
-    def namespaces():
+    def namespaces() -> Namespaces:
         ns = Namespaces()
         ns["linkml"] = URIRef("https://w3id.org/linkml/")
         ns["ex_file"] = URIRef("https://example.org/file/")
         ns["_types"] = URIRef("https://w3id.org/linkml/types/")
         return ns
 
     assert map_import(importmap, namespaces, imp) == result
+
+
+def test_parse_import_map_trailing_sep() -> None:
+    """Test the importmap namespace.
+
+    See https://github.com/linkml/linkml-runtime/issues/163.
+    """
+    importmap = parse_import_map('{ "base:": "base/" }', os.path.dirname(__file__))
+    # TODO: see how this works in a windows environment
+    assert importmap["base:"].endswith("/")
```

Comment:
was `test_issue_163.py`

I started moving tests to the appropriate files but it quickly became obvious that it would create too large a diff.